spec/index.html

```diff
@@ -812,15 +812,15 @@ <h3>Triple Terms</h3>
     </pre>
   </section>
 
-  <section id="reified-triples">
-    <h3>Reified Triples</h3>
+  <section id="reifying-triples">
+    <h3>Reifying Triples</h3>
 
-    <p>A <dfn data-cite="RDF12-CONCEPTS#dfn-reified-triple">reified triple</dfn>
+    <p>A <dfn data-cite="RDF12-CONCEPTS#dfn-reifying-triple">reifying triple</dfn>
       may be the <a data-cite="RDF12-CONCEPTS#dfn-subject">subject</a> or
       <a data-cite="RDF12-CONCEPTS#dfn-object">object</a> of an
       <a data-cite="RDF12-CONCEPTS#dfn-rdf-triple">RDF triple</a>.</p>
 
-    <p>A <a>reified triple</a> is syntactic sugar for a specific relationship between an
+    <p>A <a>reifying triple</a> is syntactic sugar for a specific relationship between an
       identifier (<dfn data-cite="RDF12-CONCEPTS#dfn-reifier">reifier</dfn>) and a <a data-cite="RDF12-CONCEPTS#dfn-triple-term">triple term</a>.
```

Comment:
I would consider this sentence as wrong now. As per the definition in PR https://github.com/w3c/rdf-concepts/pull/98, a _reifying triple_ is an RDF triple whose predicate is `rdf:reifies` and whose object is a triple term. I don't think that such an RDF triple can be said to be syntactic sugar.

Reply:
Just as `a` can be said to be syntactic sugar for `rdf:type`,
```ttl
PREFIX  ex:  <http://example.com/#>
ex:reifier  rdf:reifies    <<( ex:subject  ex:predicate  ex:object )>>
```
— can be said to be syntactic sugar for —
```turtle
PREFIX  ex:  <http://example.com/#>
ex:reifier  rdf:type       ex:triple_term ;
            rdf:subject    ex:subject ;
            rdf:predicate  ex:predicate ;
            rdf:object     ex:object .
```
Note that this does *not* include the triple `ex:subject ex:predicate ex:object` which is not itself asserted/stated; it is *only* quoted/annotated.

Reply:
Can't really say that one is syntactic sugar for the other, although we could say that the first entails the second, given appropriate semantics.

You're correct that both a not asserted or stated without also having `ex:subject ex:predicate ex:object` in the graph.